examples/src/main/python/ml/n_gram_example.py

```diff
@@ -35,7 +35,7 @@
         (2, ["Logistic", "regression", "models", "are", "neat"])
     ], ["id", "words"])
 
-    ngram = NGram(n=2, inputCol="words", outputCol="ngrams")
+    ngram = NGram(n=2, maxN=4, inputCol="words", outputCol="ngrams")
```

Comment:
This generally isn't enough to expose the new functionality in Python. Take a look in `feature.py` & also consider adding some Python tests.